### PR TITLE
translation-docusaurus-plugin-content-docs/current/troubleshooting

### PR DESCRIPTION
**Pull request by liangyongzhenya**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by RainbowMango** [comment]
@liangyongzhenya You might have to sign your commit, and here is the guideline:
https://github.com/karmada-io/website/pull/192/checks?check_run_id=8768271303

/assign @Poor12

File: i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -19,3 +19,18 @@ title: 故障定位
   ```shell
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/troubleshooting.md:19]: Hi @liangyongzhenya, would you like to translate the first case too? Great thanks:)

Comment (liangyongzhenya) [reply i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/troubleshooting.md:19]: Okay, it's been translated!

Comment (Poor12) [reply i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/troubleshooting.md:19]: BTW, please merge the commits and sign the DCO.

File: i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -2,20 +2,35 @@
 title: 故障定位
 ---
 
-## I can't access some resources when installing Karmada
+##我在安装Karmada时无法访问一些资源
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/troubleshooting.md:5]: Add a space here.

Comment (Poor12) [reply i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/troubleshooting.md]: ## 我在安装Karmada时无法访问一些资源

**Comment by Poor12** [comment]
Please squash the commits. Basiclly look good to me.

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**